src/index.tsx

```diff
@@ -452,6 +452,11 @@ export class Modalize<FlatListItem = any, SectionListItem = any> extends React.C
         toValue: destSnapPoint,
         useNativeDriver,
       }).start();
+
```

Comment:
If you check the function `onAnimateOpen` you will need to have the logic there also.

If the modal is not using `alwaysOpen`/`snapPoint` props, it won't enter in this block here. It's basically adding it where `onOpened();` is used.

It's probably better to have "duplicate" function that triggers the same result in this context, so user don't have to use 2 differents props if he wants to do only one specific thing

```
if (onPositionChange) {
  onPositionChange('top');
}
```

Reply:
This is now done @jeremybarbet and all changes pushed. Meanwhile, I discovered the case where onAnimateOpen triggers `onOpened` twice for `alwaysOpen` prop. I hope you get to look into that, or point where I can look in a new PR.

Please let me know what you think of the progress thus far